sklearn/preprocessing/tests/test_function_transformer.py

```diff
@@ -330,7 +330,7 @@ def test_function_transformer_get_feature_names_out(
     transformer = FunctionTransformer(
         feature_names_out=feature_names_out, validate=validate
     )
-    transformer.fit_transform(X)
+    transformer.fit(X)
```

Comment:
Calling `transform` allows us to find the issue with the number of columns and the function used here. Therefore, we can call `fit` to avoid this check.